### PR TITLE
Fix Typoes in application.conf

### PR DESCRIPTION
**Pull request by tuckertwo**

### CODE REVIEW

File: src/main/resources/application.conf [hunk 1]
```diff
@@ -1140,7 +1140,7 @@ opencomputers {
         "minecraft:fire"
     ]
 
-    # Whether to not show your special thinger (if you have one you know it).
```
Comment (skyem123) [inline src/main/resources/application.conf:1143]: Why not keep this? It sounds funny!

Comment (Vexatos) [inline src/main/resources/application.conf:1143]: This is, in fact, the official description for the thing described: The Special Thinger™. You mustn't change this.

Comment (fnuecke) [inline src/main/resources/application.conf:1143]: Yeah, that's very much intentional and here to stay ;-)

File: src/main/resources/application.conf [hunk 2]
```diff
@@ -282,7 +282,7 @@ opencomputers {
     # the bane of anything using a tool other than a sword or shears. This is
     # an utter pain in the part you sit on, because it makes robots meant to
     # dig holes utterly useless: the poor things couldn't break cobwebs in
-    # mining shafts with their golden pick axes. So, if this setting is true,
+    # mining shafts with their golden pick axes. So, if this setting is true,sw
```
Comment (fnuecke) [inline src/main/resources/application.conf]: Eh?

**Comment by fnuecke** [comment]
Squash please :-)

**Comment by Vexatos** [comment]
You can yourself squash while merging now.

**Comment by fnuecke** [comment]
Thanks for the reminder.

**Pull request merged**